packages/react-output-target/README.md

```diff
@@ -85,5 +85,7 @@ That's it! You can now import and use your Stencil components as React component
 | `esModule`              | If `true`, the output target will generate a separate ES module for each React component wrapper. Defaults to `false`.                                                                                                                                                         |
 | `stencilPackageName`    | The name of the package that exports the Stencil components. Defaults to the package.json detected by the Stencil compiler.                                                                                                                                                    |
 | `excludeComponents`     | An array of component tag names to exclude from the React output target. Useful if you want to prevent certain web components from being in the React library.                                                                                                                 |
-| `experimentalUseClient` | If `true`, the generated output target will include the [`use client;`](https://react.dev/reference/react/use-client) directive.                                                                                                                                               |
+
 | `customElementsDir`     | The directory where the custom elements are saved. This value is automatically detected from the Stencil configuration file for the `dist-custom-elements` output target. If you are working in an environment that uses absolute paths, consider setting this value manually. |
+
+| `hydrateModule`         | For server side rendering support, provide the package for importing the [Stencil Hydrate module](https://stenciljs.com/docs/hydrate-app#hydrate-app), e.g. `my-package/hydrate`. This will generate two files: a `component.server.ts` which defines all component wrappers and a `components.ts` that re-exports these components for importing in your application. |
```

Comment:
Sorry to be late to the party but these additions to the table aren't included in it due to the extra new lines between them and the existing table.

![image](https://github.com/user-attachments/assets/45e9e0e1-cb8f-4105-9455-1a71c909e7e6)

Reply:
@smorrisods thanks for pointing this out, mind raising a PR?

Reply:
I'd be happy to!

Reply:
Upon further investigation it looks like it has already been fixed in commit 75c6cce4cc927404335f5d400678200008ed4230! No further PRs needed 🙂.